### PR TITLE
dev-ifc2ontobim-agent-add-address

### PR DESCRIPTION
**Pull request by qhouyee**
Added the instantiation of addresses for Pirmasens and Singapore into the icontact:Address format.
Bug fix for namespace error when directly using the IfcOwl TTL file for conversion instead of the IFC model.

### CODE REVIEW

**Review by sm453 — CHANGES_REQUESTED** [review]
Could you please also revise the IfcOwlConverter readme file to account for this way of building the images? It currently says "Please build the image for this agent and then proceed to the other agent...". It should say something like "Even though you can build the image for this agent individually, there is no need to do so. You can proceed directly to the other agent...".

**Comment by sm453** [comment]
Also, it seems like the agent is _deleting_ the `ttl` file from the `data` folder after a successful upload. It would be useful to put a (big, fat) warning in the readme file that it does that. Or am I going crazy?

File: Agents/Ifc2OntoBIMAgent/README.md [hunk 1]
```diff
@@ -72,30 +75,20 @@ docker compose -f "./docker/docker-compose.test.yml" up -d --build
...
 
 **DEVELOPMENT ENVIRONMENT**
-- An image of [IfcOwlConverterAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/IfcOwlConverterAgent)
-  will need to be first created on your local Docker environment with the credentials stored.*  
-  Run the following code in the CLI at the `<root>` directory of that agent. Do note that the version numbers must match the `docker-compose.yml` files of this agent
-```
-docker build . -t ifcowlconverter-agent:1.0.0
-```
-- Deploy the agent for development by running the following code in the CLI at the directory. The remote JVM endpoint for debugging will be available at port 5005.
+- There is no need to set up the [IfcOwlConverterAgent](https://github.com/cambridge-cares/TheWorldAvatar/tree/main/Agents/IfcOwlConverterAgent) separately.
```
[the remark this answers is not in the document]
Comment (qhouyee) [reply Agents/Ifc2OntoBIMAgent/README.md]: Have updated the instructions for both agents as per comments in the latest commit

**Comment by qhouyee** [comment]
> Also, it seems like the agent is _deleting_ the `ttl` file from the `data` folder after a successful upload. It would be useful to put a (big, fat) warning in the readme file that it does that. Or am I going crazy?

Have added a warning to the README, and indicated that a TTL file for OntoBIM ABox is never generated during this process.

**Review by sm453 — APPROVED** [review]

**Pull request merged**